.buildkite/scripts/health-report-tests/main.sh

```diff
@@ -4,12 +4,11 @@ set -euo pipefail
 
 export PATH="/opt/buildkite-agent/.rbenv/bin:/opt/buildkite-agent/.pyenv/bin:/opt/buildkite-agent/.java/bin:$PATH"
 export JAVA_HOME="/opt/buildkite-agent/.java"
+export PYENV_VERSION="3.11.5"
+
 eval "$(rbenv init -)"
 eval "$(pyenv init -)"
 
-echo "--- Installing pip"
-sudo apt-get install python3-pip -y
-
 echo "--- Installing dependencies"
 python3 -mpip install -r .buildkite/scripts/health-report-tests/requirements.txt
```

Comment:
Nit: I realize python knows flags are a single letter and you dont necessarily have to have a space, but I think the space makes it much more readable. 
```suggestion
python3 -m pip install -r .buildkite/scripts/health-report-tests/requirements.txt
```